merchant-app/app/src/main/java/com/googleinterns/smb/model/Product.java

```diff
@@ -10,7 +12,7 @@
     private Double MRP;
     private Double discountedPrice;
     private String imageURL;
-    private static final String RUPEE = "\u20b9";
+    public static final String RUPEE = "\u20b9";
```

Comment:
If this package wise used, can we move it to Constants

Reply:
its only used in that class, should I move to constants?

Reply:
Then why it is moved to public ?